src/commands/snsAlerts.js

```diff
@@ -0,0 +1,44 @@
+const moment = require('moment');
+const { Sns } = require('../services');
+const { cooldown } = require('../../config/bot');
+
+module.exports = {
+  active: true,
+  args: true,
+  cooldown,
+  name: 'sns24',
+  usage: `
+      **!sns24 alerts** - *Mostra os alertas do sns.*
+    `,
+
+  /**
+  * Send to Discord SNS24 alerts.
+  *
+  * @async
+  * @param {Message} message
+  * @param {Array} args
+  */
+  async execute(message, args) {
+    if (this.args && args.length > 0) {
+      if (args[0].toLowerCase() !== 'alerts') {
+        message.reply(`Desconheço essa opção.\n${this.usage}`);
+      } else {
+        const events = [];
+        const data = await Sns.getAlerts();
+
+        if (data.length === 0) {
+          message.channel.send('***Sem Alertas***');
+
+          return;
+        }
+
+        data.forEach((item) => {
```

Comment:
Should this be a `.map()` instead of `.forEach()`? In that case we would be using a more functional approach since what you're doing there is mapping things.

```js
const events = data.map((item) => (
  `**${item.title}**\n${item.description}\n${item.link}\n${moment(item.dateTime, moment.ISO_8601)}`
));
```

Reply:
yeap, forgot the map function.